reana_commons/config.py

```diff
@@ -51,16 +51,18 @@
 OPENAPI_SPECS = {
     'reana-workflow-controller': (
         'http://{address}:{port}'.format(
-            address=os.getenv('WORKFLOW_CONTROLLER_SERVICE_HOST', '0.0.0.0'),
-            port=os.getenv('WORKFLOW_CONTROLLER_SERVICE_PORT_HTTP', '5000')),
+            address=os.getenv('REANA_WORKFLOW_CONTROLLER_SERVICE_HOST',
+                              '0.0.0.0'),
+            port=os.getenv('REANA_WORKFLOW_CONTROLLER_SERVICE_PORT_HTTP',
+                           '5000')),
         'reana_workflow_controller.json'),
     'reana-server': (
         os.getenv('REANA_SERVER_URL', 'http://0.0.0.0:80'),
         'reana_server.json'),
     'reana-job-controller': (
         'http://{address}:{port}'.format(
-            address=os.getenv('JOB_CONTROLLER_SERVICE_HOST', '0.0.0.0'),
```

Comment:
We can profit from this change and amend pytest-reana requirement boundaries in `setup.py` to:

```
'pytest-reana>=0.7.0.dev20191219,<0.8.0',
```